lua/cmake-tools/init.lua

```diff
@@ -38,6 +38,18 @@ function cmake.setup(values)
     telescope.load_extension("cmake_tools")
   end
   const = vim.tbl_deep_extend("force", const, values)
+  if const.cmake_executor.name == "terminal" then
+    const.cmake_executor = const.cmake_terminal
+    if const.cmake_notifications.enabled then
+      log.info("Cannot use cmake-tools notifications in terminal mode")
```

Comment:
Cannot use cmake-tools notifications in terminal mode -> Currently we don't support using cmake-tools notifications in terminal mode